### DISCUSSION TITLE
Any tips on debugging a "syntax error in text" error message?

### DISCUSSION THREAD

**Discussion by samsmithnz** [mermaid-js/mermaid#4301] [General]
The graph works fine in [mermaid live](https://mermaid.live/edit#pako:eNrVlV1v2jAUhv-K5atNAoITEj4ETNO6C6ShTaDdbKkqYzvBamJHtgNFiP8-xymDNqxaKyFtuYjs4-Pj87z-OHtIJGVwBJNMbskaKwO-LGIB7JdzwRT6GcMx5RuAM56KCWHCMDUd8zwFWpHJ2phCjzxPG2w46Wj71wkmRqpdm-CMlBm27Q6RuWeneCnO2feCYsMib16Fn98Hd34YdQqRfthMUBT5ERoG_RBos8vYJMcP7S2nZj1C3W7xMB2v1NTNA_N7VHXezZQU4Kti722ans1zegva7Rgezc4n6oJScKM9S2T92u0p0DnLbGKoJj32rsa6rBaoaNFraR98sKyzO-HORCrNJWA34PxQ2EAmUmijStLEvhSg92IAvw5wHvJq2n06LfIW_RA4C3DScCHpJQWt-S_0Cxr4wT-L3wkbAiyJYttn9M72p9tyBtproPv_E7rb5G92yYsPhhtwfsErRehdTYSPWrN8lbE3Hv7f0x3WgnGRSEUYrVn1MxUa425W8y4wQR_fkKp1NfaZVfiGaXJ3yqvapzq1p2IMQtQbDKIXxAjBGV1R0zn0GN5WLLAFc6ZyzKmthvvKEkOzZjmL4cg2KUtwmZkYxuJgXXFp5HInCBzZE8BasHQp33CcKpw_NX6m3ALCUYIzbY2ZxJTZ7h6aXVFV3pRrYyNa1oSnlb1UmTUfdauGOyk363LlRNKcVmV6vRlGnj0DA-wHLOoHOAwCSlZoOEj8Hkpov4t8DA-HFmRu_Xld5l21b8ECix9S5o9JHX4B_ji-4A), but crashes when trying to render with MermaidJS 10.1 in a web page.

**Comment by reid-spencer** [comment]
@samsmithnz - I feel your pain. For me the problem does happen on mermaid live and there are ZERO details to tell you WHAT the syntax error is so there's no way to find out what's wrong.
